include/mesos/mesos.proto

```diff
@@ -205,9 +205,12 @@ message MachineInfo {
  * Describes a framework.
  */
 message FrameworkInfo {
-  // Used to determine the Unix user that an executor or task should
-  // be launched as. If the user field is set to an empty string Mesos
-  // will automagically set it to the current user.
+  // Used to determine the Unix user that an executor or task should be
+  // launched as. If the user field is set to an empty string
+  // MesosSchedulerDriver will automagically set it to the current user.
+  //
+  // NOTE: The user has to be set explicitly in case the HTTP API
+  // is used.
```

Comment:
Can we restructure these two sentences as this is not very clear.

Maybe consider something among:

```
// Used to determine the Unix user that an executor or task should be
// launched as.
//
// If using HTTP API, this field must be explicitly set.
//
// If using MesosSchedulerDriver and this field is set to an empty string, MesosSchedulerDriver
// will automagically set it to the current user.